kola/tests/ignition/filesystem.go

```diff
@@ -253,6 +256,52 @@ func init() {
 		UserData:    swapConfigV2_1,
 		Distros:     []string{"cl"},
 	})
+
+	swapActivation := conf.ContainerLinuxConfig(fmt.Sprintf(`storage:
+  files:
+  - path: /etc/sysctl.d/80-swappiness.conf
+    filesystem: root
+    mode: 0644
+    contents:
+      inline: "vm.swappiness=10"
+
+systemd:
+  units:
+    - name: var-vm-swapfile1.swap
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Turn on swap
+        Requires=create-swapfile.service
+        After=create-swapfile.service
+
+        [Swap]
+        What=%s
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: create-swapfile.service
+      contents: |
+        [Unit]
+        Description=Create a swapfile
+        RequiresMountsFor=/var
+        ConditionPathExists=!%s
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/mkdir -p /var/vm
+        ExecStart=/usr/bin/fallocate -l %s %s
+        ExecStart=/usr/bin/chmod 600 %s
+        ExecStart=/usr/sbin/mkswap %s
+        RemainAfterExit=true`, swapFileName, swapFileName, swapSize, swapFileName, swapFileName, swapFileName))
+
+	register.Register(&register.Test{
+		Name:        "cl.swap_activation",
+		Run:         testSwapActivation,
+		ClusterSize: 1,
+		UserData:    swapActivation,
+		Distros:     []string{"cl"},
```

Comment:
If I understand the [discussion](https://github.com/flatcar-linux/Flatcar/issues/133#issuecomment-1025515358) correctly, the systemd unit described on the doc started working out of the box since recent versions of systemd, like v249, right?
In that case, adding a test like that would fail with LTS 2605.x.y, as LTS still has systemd v246.
Maybe restrict the test like `MinVersion:  semver.Version{Major: 3033}`?

Reply:
Good catch thanks, it should fail indeed with LTS so let's add a minimum version.